core/src/test/java/io/aiven/klaw/service/TopicSyncControllerServiceTest.java

```diff
@@ -95,6 +97,15 @@ public void setUp() throws Exception {
     loginMock();
   }
 
+  private void EnvironmentSetUp() {
```

Comment:
```suggestion
  private void environmentSetUp() {
```

Reply:
prefer to have method names starting lower case

Reply:
yip not sure how I missed that one. 100% agree

Reply:
Updated to correct casing in the latest commit.